Example/GPXKit/ViewController.swift

```diff
@@ -13,18 +13,23 @@ class ViewController: UIViewController {
     
     var tracks = [GPXTrack]()
     var waypoints = [GPXWaypoint]()
-
+    
     override func viewDidLoad() {
         super.viewDidLoad()
         
+        title = "CoreGPX"
+        
         // Example of parsing a GPX file from a sample URL
         
         let urlString : String = "https://raw.githubusercontent.com/gps-touring/sample-gpx/master/BrittanyJura/Courgenay_Ballon-DAlsace.gpx"
         let url: URL = URL(string: urlString)!
         
         // GPXRoot object that contains all the data parsed from GPXParser.
-        let gpx = GPXParser(withURL: url).parsedData()
-        
+        let timezone = TimeZone(secondsFromGMT: 0)
+        guard let gpxParser = GPXParser(withURL: url, timezone: timezone) else {
```

Comment:
For this one I specified a custom `timezone`

Reply:
The GPX Schema [here](https://www.topografix.com/gpx/1/1/gpx.xsd) specifically states this:

```
<xsd:element name="time"			type="xsd:dateTime"		minOccurs="0">
		<xsd:annotation>
		  <xsd:documentation>
			Creation/modification timestamp for element. Date and time in are in Univeral Coordinated Time (UTC), not local time! Conforms to ISO 8601 specification for date/time representation. Fractional seconds are allowed for millisecond timing in tracklogs. 
		  </xsd:documentation>
		</xsd:annotation>
	  </xsd:element>
```
As such, it should be in UTC Time, stored as , and seeming stored without UTC offset value.
Also, for speed, I did not use the default `DateFormatter` for date parsing, instead, using the class `ISO8601DateParser`, which speeds things up.

Reply:
Makes sense! Thanks for getting the relevant part in.